Tests/tx_test.py

```diff
@@ -174,3 +174,11 @@ def test_remove_hints_bug180():
     actual_path = runner(CMD + ['-o', 't1', 'n', '-f', 'cid.otf'])
     expected_path = _get_expected_path('cid_nohints.ps')
     assert differ([expected_path, actual_path, '-m', 'bin'])
+
+
+def test_long_charstring_bug444():
+    # read a CFF2 VF with a charstring longer that 65535, check output
+    actual_path = runner(CMD + ['-o', '0', '-f', 'CJK-VarTest.otf'])
+    expected_path = _get_expected_path('CJK-VarTest.txt')
+    skip = ['-s', '## Filename']
+    assert differ([expected_path, actual_path] + skip)
```

Comment:
you don't need to define `skip`

Reply:
I think I do. If I comment it out, I get:
```
    def test_long_charstring_bug444():
        # read a CFF2 VF with a charstring longer that 65535, check output
        actual_path = runner(CMD + ['-o', '0', '-f', 'CJK-VarTest.otf'])
        expected_path = _get_expected_path('CJK-VarTest.txt')
        # skip = ['-s', '## Filename']
>       assert differ([expected_path, actual_path] + skip)
E       NameError: global name 'skip' is not defined
```

Reply:
sorry what I meant was, just do
```python
assert differ([expected_path, actual_path, '-s', '## Filename'])
```